README.md

```diff
@@ -1224,20 +1221,13 @@ abstract class Shape
 
 class Rectangle extends Shape
 {
-    public function __construct()
-    {
-        parent::__construct();
-        $this->width = 0;
-        $this->height = 0;
-    }
+    private $width;
+    private $height;
 
-    public function setWidth($width)
+    public function __construct($width, $height)
     {
+        parent::__construct();
```

Comment:
What is this needed for?

Reply:
In this context is not really needed because we don't have an explicitly declared constructor in the parent class, but it is a good practice to add it in case the parent defines logic in a default constructor, because php does not automatically calls parent constructors. See note in: http://php.net/manual/en/language.oop5.decon.php
But, as you have noticed, it is not really needed.

Reply:
I asked before this gives me feeling there actually is something in parent constructor.
As in most cases in practise, e.g. Symfony Command class.

Reply:
Sure